source/frontend/src/components/common/form/InputGroup.tsx

```diff
@@ -77,6 +77,7 @@ export const InputGroup: React.FC<React.PropsWithChildren<InputGroupProps>> = ({
         'flex-nowrap',
         'input-group',
       )}
+      noGutters
```

Comment:
this fixes alignment issues with the selectinput component:

![image](https://github.com/user-attachments/assets/b88cbf4e-efcb-4e50-a657-fa6e84bed333)